src/index.ts

```diff
@@ -34,4 +34,7 @@ https://firebase.google.com/docs/web/setup
   console.error(message);
 }
 
+//export {FirebaseNamespace} from './firebase-namespace';
```

Comment:
I'd expect this file to become the barrel export for this module at some point.

Reply:
I agree. Right now I put it in `default-namespace.ts` but I think like my other comment it should be just one file (in `index.ts`)